File: live_streaming/app/src/main/java/com/zegocloud/uikit/livestreaming/LiveActivity.java
package com.zegocloud.uikit.livestreaming;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.zegocloud.uikit.prebuilt.livestreaming.ZegoUIKitPrebuiltLiveStreamingConfig;
import com.zegocloud.uikit.prebuilt.livestreaming.ZegoUIKitPrebuiltLiveStreamingFragment;

public class LiveActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_live);

        addFragment();
    }

    private void addFragment() {
        long appID = getIntent().getLongExtra("appID", 0L);
        String appSign = getIntent().getStringExtra("appSign");
        String userID = getIntent().getStringExtra("userID");
        String userName = getIntent().getStringExtra("userName");

        boolean isHost = getIntent().getBooleanExtra("host", false);
        String liveID = getIntent().getStringExtra("liveID");
//        long appID = YourAppID;
//        String appSign = YourAppSign;
//
//        String liveID = yourLiveID;
//        String userID = YourUserID;
//        String userName = YourUserName;

        ZegoUIKitPrebuiltLiveStreamingConfig config;
        if (isHost) {
            config = ZegoUIKitPrebuiltLiveStreamingConfig.host();
        } else {
            config =  ZegoUIKitPrebuiltLiveStreamingConfig.audience();;
        }

        ZegoUIKitPrebuiltLiveStreamingFragment fragment = ZegoUIKitPrebuiltLiveStreamingFragment.newInstance(
            appID, appSign, userID, userName, liveID, config);
        getSupportFragmentManager().beginTransaction()
            .replace(R.id.fragment_container, fragment)
            .commitNow();
    }
}

File: live_streaming_compose/app/src/main/java/com/zegocloud/demo/myapplication/ui/LiveActivity.kt
package com.zegocloud.demo.myapplication.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.zegocloud.demo.myapplication.R
import com.zegocloud.uikit.prebuilt.livestreaming.ZegoUIKitPrebuiltLiveStreamingConfig
import com.zegocloud.uikit.prebuilt.livestreaming.ZegoUIKitPrebuiltLiveStreamingFragment

class LiveActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_live)

        addFragment()
    }


    private fun addFragment() {
        val appID =
        val appSign =
        val userID = "123324"
        val userName = "235345"
        val isHost = true
        val liveID = "345345"
        val config = if (isHost) {
            ZegoUIKitPrebuiltLiveStreamingConfig.host(true)
        } else {
            ZegoUIKitPrebuiltLiveStreamingConfig.audience(true)
        }
        val fragment: ZegoUIKitPrebuiltLiveStreamingFragment =
            ZegoUIKitPrebuiltLiveStreamingFragment.newInstance(appID, appSign, userID, userName, liveID, config)
        supportFragmentManager.beginTransaction().replace(R.id.fragment_container, fragment).commitNow()
    }
}